src/wp-admin/includes/class-walker-nav-menu-edit.php

```diff
@@ -262,6 +262,29 @@ public function start_el( &$output, $data_object, $depth = 0, $args = null, $cur
 					<button type="button" class="button-link menus-move menus-move-top" data-dir="top"><?php _e( 'To the top' ); ?></button>
 				</fieldset>
 
+				<?php
+				/**
+				 * Update parent and order of menu item by changing it.
+				 *
+				 * @since 6.6.0
+				 */
+				?>
+
+				<fieldset class="field-move-combo description description-wide">
+					<div class="" style="display: flex;">
```

Comment:
To inherit related styles, use `p` instead of `div` and add the class 'description description-wide'.